examples/websocket-retry-client/websocket_retry_client.bal

```diff
@@ -5,7 +5,10 @@ public function main() returns error? {
     websocket:Client wsClient = check new("ws://localhost:9090/foo", {
         // Set the maximum retry count to 5 so that it will try 5 times with the interval of
         // 1 second in between the retry attempts.
-        retryConfig: { maxCount: 5, interval: 1 }
+        retryConfig: {
+            maxCount: 5,
+            interval: 1
```

Comment:
shall we make the interval 5 seconds.

Reply:
Fixed with [08ef097](https://github.com/ballerina-platform/ballerina-distribution/pull/3559/commits/08ef0971d5918bbf5c48ed68414ff5006843425b)